MinimalCirclesPath.py

```diff
@@ -64,33 +64,30 @@ def getDistance(startLat, startLong, destLat, destLong):
 
 # pathWidth is the desired scaned distance from play diameter, radius is radius of circle passed to NOTAM
 def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
-
-    # TODO For actual application you would need to capture N/S/E/W
-
     
-      
-    # gets total distance from start to finish
-    totalDistance = getDistance(startLat, startLong, destLat, destLong)
-
     # gets step distance (pythagorean theorum)
     stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
 
     # gets direction
     bearing = calculateBearing(startLat, startLong, destLat, destLong)
 
+    # updates the start so the area in the opposite direction of the path is right on the edge of the first circle
+    updatedStart = nextPoint(startLat,startLong, bearing, radius - (pathWidth/2) )
+
+    # updates the dest so that it is the furthest point on the path. This means going the pathWidth/2 past the dest
+    updatedDest = nextPoint(destLat, destLong, bearing, (pathWidth/2))
```

Comment:
Is `updatedDest` only used for distance calculation and hence the number of "nextPoints" that will be appended to the `coordList`?  I see you removed adding `destLat,destLong` as the final coordinate in the `coordList`.  Are we sure that the path of coordinates ends on `updatedDest`?  Or should it?

Reply:
It is only used for the distance calculation. The reason I removed it is because we don't need to call the API from the furthest point we only need to have it within the radius. This is Also why I need to update it so it some area past the airport.

Reply:
does this mean the final call to the NOTAMs API only guarantees that the destination airport is within the radius of that final call, but makes no guarantee of how much further beyond the destination airport the program may fetch NOTAMs for?

Reply:
It guarantees you get points at least pathWidth/2 past the destination airport. This is the why I update the dest before getting totalDistance. However, it may fetch more than that if the rest of the path distance isn't split up evenly